assets/samples/cloudNativeFullStack.yaml

```diff
@@ -135,35 +155,54 @@ spec:
       #     cpu: 300m
       #     memory: 1.5Gi
 
+      # Optional: The URI of the image that contains the codemodules specific OneAgent that will be injected into pods and applications.
+      # For an example of a Dockerfile creating such an image, see https://dt-url.net/operator-docker-samples
+      #
+      # codeModulesImage: ""
+
   # Configuration for ActiveGate instances.
+  #
   activeGate:
-    # Enables listed ActiveGate capabilities
+    # Specifies which capabilities will be enabled on ActiveGate instances
+    # The following capabilities can be set:
+    # - routing
+    # - kubernetes-monitoring
+    # - metrics-ingest
+    # - dynatrace-api
+    #
     capabilities:
       - routing
       - kubernetes-monitoring
       - dynatrace-api
 
-    # Optional: to use a custom ActiveGate Docker image.
-    image: ""
-
-    # Amount of replicas of ActiveGate pods
+    # Optional: Sets how many ActiveGate pods are spawned by the StatefulSet
+    # Defaults to "1"
     #
     # replicas: 1
 
-    # Optional: tolerations to include with the ActiveGate StatefulSet.
-    # See more here: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    # Optional: Sets the image used to deploy ActiveGate instances
+    # Defaults to the latest ActiveGate image on the tenant's registry
```

Comment:
Nitpick: what's the tenant's registry?

"Defaults to the latest ActiveGate image available for the tenant."

Reply:
the tenants registry is the image registry built into a tenants environment. I guess if a specific AG version is available on a tenant, so is probably it's image, but this way would be a bit more specific that it is indeed an image we use and not just any binary.

Reply:
Maybe an example for an valid image path might be helpful. here. If I'd as a operator user wanted to add something useful to this field, I'd be left with questions like "where's this registry?", "what's the correct image path?" and "where do I get all this infos from?"

Reply:
Like the idea, something like `<tenanturl>/linux/activegate:latest` so everyone knows where to pull it from

Reply:
done